cmd/osbuild-image-tests/context-managers.go

```diff
@@ -90,7 +91,25 @@ func withBootedQemuImage(image string, ns netNS, f func() error) error {
 			return fmt.Errorf("cannot close temporary cloudinit file: %#v", err)
 		}
 
-		qemuCmd := ns.NamespacedCommand(
+		var qemuCmd *exec.Cmd
+		// This command does not use KVM as I was unable to make it work in Beaker,
+		// once we have machines that can use KVM, enable it to make it faster
+		qemuAarch64Cmd := ns.NamespacedCommand(
+			"qemu-system-aarch64",
+			"-cpu", "host",
+			"-M", "virt",
+			"-m", "2048",
+			// As opposed to x86_64, aarch64 uses UEFI, this one comes from edk2-aarch64 package on Fedora
```

Comment:
I think we should add a dependency to that package to the spec file.

Reply:
good idea, fixed